tensorboard/webapp/experiments/types.ts

```diff
@@ -12,7 +12,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-export interface Experiment {
+export declare interface Experiment {
```

Comment:
Why does `Experiment` need to be declared non-mangleable / where Is it serialized?

Reply:
Sorry, I should've read the description closely.  It's a backend response, so we'd like to ensure we can read properties off it.